.github/workflows/doc-lint-test.yaml

```diff
@@ -246,6 +319,14 @@ jobs:
             echo "::error::Building html docs ${ret##*:}"
             exit 1
           fi
+          echo "BUILT_DOCS_PATH=$PWD/build/sphinx/html" >> $GITHUB_OUTPUT
+      - name: Upload HTML Docs
+        uses: actions/upload-artifact@v4
+        with:
+          name: HTML Docs
+          path: ${{ steps.build-html-docs.outputs.BUILT_DOCS_PATH }}
+          if-no-files-found: error
+          overwrite: true
```

Comment:
Where is this putting build/sphinx/html?

Reply:
Just an artifact for reference purposes?  Not used?

Reply:
Yep, this just becomes an artifact. It can be downloaded if someone wants to inspect the result but otherwise isn't used.